### PR TITLE
works with avatar

### PR DESCRIPTION
**Pull request by LiliiaShemet**
Будь ласка, перевірте.  І на постмані .

### CODE REVIEW

File: routes/api/users.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ const express = require("express");
 const auth = require("../../middlewares");
 const ctrl = require("../../controllers/users");
 const router = express.Router();
+const upload = require("../../middlewares/upload");
+const { userControllers } = require("../../models/user");
```
Comment (Prykhodko-Pasha) [inline routes/api/users.js]: UserControllers doesn't exist in /models/user.js

File: controllers/userControllers [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    const fileUpload = path.join( avatarsDir, newFilename);
+    await fs.rename(tempUpload, fileUpload);
+    const avatarURL = path.join("avatars", newFilename);
+    await User.findByIdAndUpdate(_id, { avatarURL });
+    res.json({ avatarURL });
+  } catch (err) {
+    next(err);
+  }
+};
+
```
Comment (Prykhodko-Pasha) [inline controllers/userControllers]: Please rename this file according to the other files in this folder

File: controllers/users/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ const logoutUser = require("./logout");
 const verifyUserEmail = require("./verifyEmail");
 const sendVerifLetter = require("./sendVerifLetter");
 
+
```
Comment (Prykhodko-Pasha) [inline controllers/users/index.js:9]: And do re-export of your controller here

Comment (LiliiaShemet) [reply controllers/users/index.js:9]: Зробила

File: controllers/userControllers (hunk 2 above)
Comment (LiliiaShemet) [reply controllers/userControllers]: перейменувала

File: routes/api/users.js (hunk 1 above)
Comment (LiliiaShemet) [reply routes/api/users.js]: ок

Comment (LiliiaShemet) [reply routes/api/users.js]: ок

**Review by Prykhodko-Pasha — APPROVED** [review]

**Review by SovaV — APPROVED** [review]

**Pull request merged**